### ISSUE TITLE
Analytics and Adsmanager shows wrong revenue by 10x+ for Android only(iOS data is correct)

### ISSUE THREAD

**Issue by Kendja** [facebook/facebook-sdk-for-unity#232]
Hi.

I use default conversion events.
Facebook analytics shows me crazy revenue numbers for Android by 10x and more. But iOS data is correct.

I also run facebook ads to acquire users for app actions. And revenue for Android shows me 2x than it is. 
For iOS also everything is correct here.

I use Firebase in the game. Could there be a conflict of some sort that it sends event value multiple times?

Facebook-sdk-for-unity: 7.15.1
Unity version: 2018.3.3f1

**Comment by KylinChang** [comment]
@Kendja , thanks for your report. Can you report the bug in https://developers.facebook.com/support/bugs/?

**Comment by tayl0r** [comment]
I'm seeing the same thing, the Purchase event (and all the events are duplicated by 5-6x every time). I would assume this is the reason for the significantly higher revenue numbers.

![image](https://user-images.githubusercontent.com/323663/56784904-bd9ea180-67a7-11e9-8f14-935314edd0d9.png)

**Comment by Kendja** [comment]
@KylinChang oops, missed your reply. 
I sent a bug report via facebook

**Comment by Kendja** [comment]
@KylinChang Got a reply from Facebook team, that it it's fixed. But I still see wrong numbers in analytics.

And I don't see any related commits for this issue.

https://developers.facebook.com/support/bugs/515260065544773/

any ideas?

**Comment by eladleb** [comment]
I'm experiencing the same issue as well with v7.16.1 :(

**Comment by Kendja** [comment]
Updated to v7.17.1. Now revenue shows wrongly by 10x+ up to 20x+.
Using Unity 2019.1.8f1.

@KylinChang @dreamolight my bug report on https://developers.facebook.com/ was closed as resolved. And they don't reply anymore there. 
Please, check it out.

**Comment by bobob89** [comment]
I have same problem. Last facebook sdk and Unity 2019.1.8f1.

**Comment by andreyfaraponov** [comment]
have the same problems,
Facebook SDK 7.17.1

**Comment by eladleb** [comment]
I am experiencing a similar issue but only for iOS, using 7.17.2 @KylinChang is anyone from your team looking at this...? It makes the SDK unusable.
Every other analytics product we have reports accurate numbers except for Facebook's Unity SDK :(

**Comment by Kendja** [comment]
@bobob89 @andreyfaraponov @eladleb I submitted another bug report via facebook:
https://developers.facebook.com/support/bugs/391897685015143/

If more people will write there I bet it will help :)

**Comment by KylinChang** [comment]
Hey @eladleb @Kendja , do you use the latest Unity SDK? Have you tried other versions of Unity SDK?

**Comment by eladleb** [comment]
@KylinChang Yes, I'm using the latest unity sdk. We suspect the false iOS revenue numbers are coming from users who are using a pirated version of the game as it seems that there is no server side receipt validation when reporting Facebook Purchases?

**Comment by Kendja** [comment]
@KylinChang yep. Both different versions of unity and different versions of this plugin has been used since April

**Comment by drallcom3** [comment]
v7.18.0 The issue still persists. Bug reports about it on https://developers.facebook.com/support/bugs/ get deleted or marked as "fixed", which it clearly isn't.

**Comment by Kendja** [comment]
Same. But now also iOS revenue is shown wrong by 2~3x times

Unity 2019.2.11f1
vFb 7.18.0

**Comment by bobob89** [comment]
Still have it too.

**Comment by RAEadmin** [comment]
> I'm seeing a similar thing thing, the Purchase event (and all the events are duplicated by 5-6x every time). I would assume this is the reason for the significantly higher revenue numbers.
> 
> This is not a new bug. I tested with both the latest SDK, 7.15.1, and our old version, 7.13.0. Both had the same problem. I also reported the bug via FB Developers site.
> 
> I'm on Unity 2017.4 LTS
> 
> ![image](https://user-images.githubusercontent.com/323663/56784904-bd9ea180-67a7-11e9-8f14-935314edd0d9.png)

@tayl0r did you need a specific permission to view the analytics debug? I'm an admin on the facebook dev console roles, am logged in to the personal facebook account on ios facebook app on device, but I still do not see the events appear in the debug on Events Manager? Did you need to add any code for the Events Testing to work please?

**Comment by mehves-veloxia** [comment]
Still experiencing this issue, it is tracking **both iOS and Android** purchases wrong by **~10x+** for both platforms.

SDK v7.19.2 (latest now)
Unity 2019.2.21

**Comment by dlees** [comment]
I'm seeing the same problem after upgrading from Unity 2019.8.3. Only my iOS purchases are duplicated. I don't post the events myself (turn off auto revenue tracking results in 0 revenue shown).

SDK: 9.0.0 
Unity:2019.4.11

Has there been any fix for this?

**Comment by mastef** [comment]
Is there a guide on how to disable the broken auto inapp tracking with manual inapp tracking?
